src/tests/system/tests/test_passkey.py

```diff
@@ -8,6 +8,77 @@
 or even ssh<user>@localhost.
 Here, passkey support is tested with su, tests are running with
 umockdev, not with a physical key.
+
+we use sssctl tool to create the passkey-mapping or token.
+# sssctl passkey-register --username=<username> --domain=<domain name>
+Next, it will ask for PIN and generate the passkey-mapping and token.
+
+for example:
+    [root@client ~]# sssctl passkey-register --username=joe --domain=ipa.test
+    Enter PIN:
+
+    Please touch the device.
+    passkey:N5YdS4ZGLS6v7BWYSrvVygRPsvtqTJ9DrAclM5S1C/18axs8XutDEYuKiQrZGwOEpxGsHE1q19A4OOi0bct29g==,MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEX/gpbrCR/Q+f3CQE9GjVj7Ck+uHl9x5HuacZ+xitqpanwmRzqdu6ls2/CmaOuPs29Sgpi/HZXexW2R5okV4Stg==
+
+the output string has `PASSKEY:credentialId,publicKey` format.
+
+The following code will add the user with passkey-mapping which was created while creating the umockdev.script and
+umockdev.ioctl file using the umockdev-run command. Here, testdatadir is 'sssd/src/tests/system/data
+/test_passkey/<test_method_name>'
+.. code-block::
+    with open(f"{testdatadir}/passkey-mapping.{suffix}") as f:
+        provider.user("user1").add().passkey_add(f.read().strip())
+
```

Comment:
I forgot about https://tests.sssd.io/en/latest/guides/testing-passkey.html , I think we should update the description in the test framework code, everything above line 23.